florist/tests/integration/api/test_train.py

```diff
@@ -64,7 +64,7 @@ def test_train():
                 client_uuid = response.json()["client_uuids"][0]
 
                 # Wait for training to finish
-                wait_for_metric(server_uuid, "fit_end", test_redis_host, test_redis_port, LOGGER, max_retries=60)
+                wait_for_metric(server_uuid, "fit_end", test_redis_host,test_redis_port, LOGGER, max_retries=80)
```

Comment:
Any reason why the max retries have been increased to 80 here? Was it failing with 60? Also, you should restore the space removed before the comma.

Reply:
Yeah it was failing on my local with 60 so I bumped to 80! But I will restore the space :)